doc/guides/SimpleStartupGuide.md

```diff
@@ -5,7 +5,7 @@
 [System Requirements](#system-requirements)  
```

Comment:
Similar themes in this doc as compared to CustomStartupGuide.md. Not repeating here since comments in the other doc.